boards/nucleo-f767zi/include/periph_conf.h

```diff
@@ -126,6 +126,38 @@ static const uart_conf_t uart_config[] = {
     }
 };
 
+/**
+ * @name ETH configuration
+ * @{
+ */
+#define ETH_NUMOF           (1)
+#define ETH_RX_BUFFER_COUNT (4)
+#define ETH_TX_BUFFER_COUNT (4)
+ #define ETH_RX_BUFFER_SIZE (1524)
+#define ETH_TX_BUFFER_SIZE (1524)
+ #define ETH_DMA_ISR        isr_dma2_stream0
+ static const eth_conf_t eth_config = {
+    .mode = RMII,
+    .mac = { 0 },
+    .speed = ETH_SPEED_100TX_FD,
+    .dma_chan = 0,
+    .dma_stream = 8,
+    .phy_addr = 0x01,
+    .pins = {
+        GPIO_PIN(PORT_G, 13),
```

Comment:
New Pinout for F767ZI. I would assume this is common for all F7 devices. I will check. If my assumption is right this should move to f7 common, right?

Reply:
Same for all f7 devices but I will let it in place. I think it is unnecessary overhead for all non ethernet boards.